File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/Simplace_SoilTemperature/src/cs/Simplace_SoilTemperature/SoilTemperatureAuxiliary.cs
using System;
using System.Collections.Generic;

public class SoilTemperatureAuxiliary 
{
    private double _SnowIsolationIndex;
    
    public SoilTemperatureAuxiliary() { }
    
    
    public SoilTemperatureAuxiliary(SoilTemperatureAuxiliary toCopy, bool copyAll) // copy constructor 
    {
    if (copyAll)
    {
    
    _SnowIsolationIndex = toCopy._SnowIsolationIndex;
    }
    }
    public double SnowIsolationIndex
        {
            get { return this._SnowIsolationIndex; }
            set { this._SnowIsolationIndex= value; } 
        }
}

File: src/pycropml/transpiler/antlr_py/tests/examples/SimplaceComponent/Simplace_SoilTemperature/src/java/Simplace_SoilTemperature/SoiltemperatureAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class SoilTemperatureAuxiliary
{
    private Double SnowIsolationIndex;
    
    public SoilTemperatureAuxiliary() { }
    
    public SoilTemperatureAuxiliary(SoilTemperatureAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.SnowIsolationIndex = toCopy.getSnowIsolationIndex();
        }
    }
    public Double getSnowIsolationIndex()
    { return SnowIsolationIndex; }

    public void setSnowIsolationIndex(Double _SnowIsolationIndex)
    { this.SnowIsolationIndex= _SnowIsolationIndex; } 
    
}